README.md

```diff
@@ -2,9 +2,95 @@
 
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
```

Comment:
Should we add some top-level navigation for MicroCeph here? It could include links to the documentation site as well as deeper links into the readme

Reply:
Aaaah, I always forget top level index in my markdowns. Ack added.